packages/avisynthplus.rb

```diff
@@ -1,37 +1,25 @@
-require 'package'
+require 'buildsystems/cmake'
 
-class Avisynthplus < Package
+class Avisynthplus < CMake
   description 'An improved version of the AviSynth frameserver'
   homepage 'https://avs-plus.net/'
-  version '3.7.2'
+  version '3.7.3'
   license 'GPL-2 and GPL-2-with-linking-exception'
-  compatibility 'all'
-  source_url 'https://github.com/AviSynth/AviSynthPlus/archive/v3.7.2/avisynthplus-3.7.2.tar.gz'
-  source_sha256 '6159fd976dffa62d5db5277cbb0b3b7f7a4ee92fc8667edd32da9840a669ccc1'
+  compatibility 'x86_64 aarch64 armv7l'
+  source_url 'https://github.com/AviSynth/AviSynthPlus.git'
+  git_hashtag "v#{version}"
   binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '33a07e40e5de09d4b6a3c932c3a7c05d9e809bf388950ed69d2127d24a7d5ac5',
-     armv7l: '33a07e40e5de09d4b6a3c932c3a7c05d9e809bf388950ed69d2127d24a7d5ac5',
-       i686: 'e97386b6df6772e0e495f29260091fac407cb50203ae9b0e41ecf9e94458a047',
-     x86_64: '122650ad7a858af46806c779da6e045c43bc267a004b2de255b947def1d4cff5'
+    aarch64: 'b9e247846b433bd2e1767f268bf34a22860e0a18a29e45528aabf8b32725f6e2',
+     armv7l: 'b9e247846b433bd2e1767f268bf34a22860e0a18a29e45528aabf8b32725f6e2',
+     x86_64: '2e7cd1267ae94d155abc1e9890f484a08a8eeb0ea75473d4875431316ae23471'
   })
 
-  depends_on 'devil' => :build
+  depends_on 'devil' # R
+  depends_on 'gcc_lib' # R
+  depends_on 'glibc' # R
+  git_fetchtags
 
-  def self.build
-    Dir.mkdir 'builddir'
-    Dir.chdir 'builddir' do
-      system "cmake \
-        -G Ninja \
-        #{CREW_CMAKE_OPTIONS} \
-        -DBUILD_SHARED_LIBS=ON \
-        .."
-    end
-    system 'ninja -C builddir'
-  end
-
-  def self.install
-    system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
-  end
+  cmake_options "-DCMAKE_BUILD_TYPE:STRING='None'"
```

Comment:
Shouldn't this be `Release`?

Reply:
I copied from here:

https://gitlab.archlinux.org/archlinux/packaging/packages/avisynthplus/-/blob/main/PKGBUILD?ref_type=heads